src/porepy/models/constitutive_laws.py

```diff
@@ -3094,7 +3094,7 @@ def gravity_force(
             material: Name of the material. Could be either "fluid" or "solid".
 
         Returns:
-            Cell-wise nd-vector representing the gravity force.
+            Cell-wise nd-vector representing the gravity force [kg*s^-2*m^-(nd-1)].
```

Comment:
Can we talk about densities for nd<3, please?

Reply:
Decision: This should be m^-2, not m^-(nd-1). I will update tutorials/conventions.ipynb.